docs/en/enterprise-edition/content-collections/application-security/get-started/connect-code-and-build-providers/ci-cd-runs/add-azure-pipelines.adoc

```diff
@@ -0,0 +1,99 @@
+:topic_type: task
+
+[.task]
+== Connect Azure Pipelines
+
+Integrate Prisma Cloud Code Security with Azure Pipelines to seamlessly embed vulnerability detection into your Azure DevOps CI/CD pipelines. This integration enables continuous scanning and detection of your your infrastructure as code (IaC) and code repositories as a part of your CI/CD workflows. Additionally, it automates shift-left actions and provides reporting and tracking on the Prisma Cloud administrative console.
+
+[.procedure]
+
+. Before you begin, xref:../../../../administration/create-access-keys.adoc[generate and copy the Prisma Cloud access key] to enable access to Prisma Cloud. The access key includes a Key ID and Secret.
+. Create or open the `.gitlab-ci.yml` file in your repository for editing, or alternatively, navigate to *CI/CD* > *Editor* in your GitLab project.
+. In *Azure DevOps*, create a new pipeline or select an existing pipeline and select *Edit*. 
++
+An editor for the `azure-pipelines.yml` configuration file is displayed.
++
+image::application-security/az-pipelines-1.png[]
```

Comment:
We'll need to add this image: https://github.com/PaloAltoNetworks/prisma-cloud-docs/blob/maxwell-fixes/code-security/admin_guide/_graphics/az-pipelines-1.png

Or delete this line. It's not critical.

Reply:
The image in the adoc seems to be the same image that you are referring to .

Reply:
You're right, I missed it